### ISSUE TITLE
Changing zoom level breaks layout

### ISSUE THREAD

**Issue by jochenberger** [deitch/jstree-grid#203]
Open http://jsfiddle.net/5qcsth7j/, scroll down a bit, select a node (e.g. Child 40), press `Ctrl`+`-`. The cells are not aligned anymore. The effect seems to be worse in Chrome than in Firefox.
![screenshot](https://cloud.githubusercontent.com/assets/370733/21103537/2fefcb0c-c083-11e6-8a4c-222756e29d3f.png)

**Comment by deitch** [comment]
And not a problem at all in Safari. Interesting.

**Comment by deitch** [comment]
I take it back. I cannot reproduce it even in Chrome. Does platform matter?

**Comment by deitch** [comment]
OK, I can reproduce it, but not in Safari (only Chrome and Brave, for now). And, more interestingly, it seems to open only at 1 step below actual size. 2 below or any above is fine. Any ideas?

**Comment by jochenberger** [comment]
I'm inclined to see it as a Chrome bug.

**Comment by deitch** [comment]
If we knew exactly what triggered it and could reproduce it outside of jstreegrid, we could open a bug with them.

A day doesn't go by with this project that I don't miss the ease of horizontal alignment that came with having the cells as sub-elements of the `li`.... and then I look back at all of the issues and remember the pain of the vertical alignment.

In truth, this should just be part of the tree itself...

**Comment by peternoges** [comment]
@deitch, days go by.  That is something we cannot stop.  However, was hoping you could catch me up on the problem here, because I bet there is a client-side trick we can adopt to make this problem go away. I'm just not sure where to start. Thanks.

**Comment by peternoges** [comment]
@deitch @jochenberger  
I know this is an old thread, but I have the same problem.  Zoom works fine until i zoom past the point at which jstree's responsive theme kicks in.  Then alignment breaks.

**Comment by deitch** [comment]
it is an old thread. Not much in it. Happy to take PRs if someone can figure it out.

**Comment by peternoges** [comment]
@deitch @jochenberger  Here are some findings:
- I reproduced the issue in my project;  

- The problem surfaces only when jstree's "responsive" variable is ENABLED.  When jstree Responsive is enabled,  it takes some patience to reproduce the issue.  Specifically jstreegrid initializes/renders itself fine irrespective of Zoom. In fact we must zoom **after** jstreegrid is initialized to see the issue (for me it's around 90% in Chrome browser). 

@deitch  The problem seems to be a compatibility issue between jstree's built-in responsive theme and jstreegrid. I suspect we can fix this easily.  Could we wrap jstreegrid's initialization code in a function that can be called in response to viewport changes, or jstreegrid css values?  Basically given the observations above it seems we just need to "re-create jstreegrid" whenever zoom level changes, or at least changes to the css/media query values in 
https://github.com/vakata/jstree/blob/master/src/themes/responsive.less 

Has anyone considered this yet?

Note I'm using jstree/3.3.5  &&  jstreegrid/3.9.4

**Comment by peternoges** [comment]
UPDATE  I am wrong.  Even with responsive OFF, I get the issue. The search continues.  Disregard https://github.com/deitch/jstree-grid/issues/203#issuecomment-449440135

**Comment by peternoges** [comment]
Update: I solved the same issue by changing two of my jstree settings:  I removed "variant": "large" from the jstree definition **and** set the jstree "responsive" variable to false.  This could be a workaround until jstreegrid can be updated (somehow?) to respond well to jstree "theme" "variants" and jstree's built-in  (but toggle-able) responsiveness. 

 Now zooming does not break the alignment.  Previously the alignment broke both on some zoom-levels, including but not limited to the level by which jstree kicks in its built-in responsiveness. 

using jstree/3.3.5  &&  jstreegrid/3.9.4
